nixos/modules/security/gnupg.nix

```
@@ -0,0 +1,253 @@
+{ config, lib, pkgs, ... }:
+let
+  inherit (builtins) dirOf match split;
+  inherit (lib) types;
+  cfg = config.security.gnupg;
+  gnupgHome = "/var/lib/gnupg";
+  escapeUnitName = name:
+    lib.concatMapStrings (s: if lib.isList s then "-" else s)
+    (split "[^a-zA-Z0-9_.\\-]+" name);
+in
+{
+options.security.gnupg = {
+  store = lib.mkOption {
+    type = types.path;
+    default = lib.maybeEnv "PASSWORD_STORE_DIR" ".password-store";
```

I would not rely on environment variables here. At least not at evaluation time. With flakes
we are trying very hard to get away from impure evaluation as it makes evaluation between different systems harder:

```suggestion
    default =  "/root/.password-store";
```

---

Thanks. I've fixed as you've suggested and just mentioned in the `description` that `getEnv PASSWORD_STORE_DIR` is a possible setting.